### DISCUSSION TITLE
Free Turbine Torque

### DISCUSSION THREAD

**Discussion by TimSkytraveler** [nguyenquyhy/Flight-Tracker-StreamDeck#328] [Ideas]
Hi, 

With the Twin Otter now on MSFS, I miss the TURB_ENG_FREE_TURBINE_TORQUE:index value for Streamdeck. 
Any chance this will be added in Flight Tracker?

regards
Tim

**Comment by rmroc451** [comment]
are you on the .39 release?


with .39 release there is no longer a "white list" of allowed. If simconnect has it, and you type it in, it will bring back the value.

You may need to specify the unit you want it in, but thats minor. :)

send a screenshot of your button config and attach your logs if you are having issues

**Reply by nguyenquyhy** [reply]
Oh I removed the requirement for adding events into the list, but variables will still need to get added 😛.

**Reply by nguyenquyhy** [reply]
Free input of variables will probably come after I can fix some issues with that.

**Reply by rmroc451** [reply]
> Free input of variables will probably come after I can fix some issues with that.

wellllllll, thats what I get for assuming. My apologies, should have double checked that lol

**Reply by TimSkytraveler** [reply]
For me that sounds like good news 😆

**Reply by Gert57** [reply]
For me also. I try to get feedback from newer airplanes, e.g. kodiak. The developer helped me with the simvars they use, but I could not get them to work, e.g. fuel flow in pounds. tried a lot, but now I understand! I wait for the next update.
Team, all thanks for the fantastic work!!

**Reply by nguyenquyhy** [reply]
For now please give me the SimVar you need. I'm pretty slow with big features 😛.

**Comment by TimSkytraveler** [comment]
Hi, yes I have the .39 release. The return I get is "0" and not an error, so I believe it might just work, but the Twin Otter (Aerosoft) probably doesn't use the TURB_ENG_FREE_TURBINE_TORQUE;index value. I'll see and ask them what they use.

Thanks for the reply

**Reply by nguyenquyhy** [reply]
`TURB ENG FREE TURBINE TORQUE:index` works well in the Twotter so I will add it in v0.40 (won't work in 0.39). Do you happen to know what unit it should use? And which gauge in the Twin Otter shows that value?

**Reply by TimSkytraveler** [reply]
The top two gauges show Torque Press.
I think it is psi, it doesn’t say. [edit: it in fact is pound-force per square foot]
The gauge is 0-75 with a red line at 50.

the P&W PT-6 works with psi torque values as far as I know [edit: it in fact is pound-force per square foot]

![image png c871781c03a71207d08de9b8f6ec18d4](https://user-images.githubusercontent.com/98173543/150609380-72d0b3b8-8a5a-43b1-bc4d-9229272ef791.png)

**Reply by TimSkytraveler** [reply]
OK, so the developer states, that the unit is pound-force per square foot

**Reply by rmroc451** [reply]
> OK, so the developer states, that the unit is pound-force per square foot
> 
> 

In the unit on the button then, type `psf` and leave focus on it, and it should update to match the sim!

**Reply by TimSkytraveler** [reply]
Ok,  but I'll have to wait for .40 as the TURB ENG FREE TURBINE TORQUE:index is not implemented in Flight Tracker, yet.
Got it.

**Reply by rmroc451** [reply]
Ah sure. If you know how to compile it, you can add it and have it now.

If not? I can send you a googledrive link to a debug version with it in there so you can try?

**Reply by TimSkytraveler** [reply]
I'm afraid I am not familiar enough with compiling code in general. Would love to get that link!

**Reply by nguyenquyhy** [reply]
0.40 is now in GitHub Releases (https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/releases). Please remember to set the unit as `psf` as suggested above 😄.

**Reply by TimSkytraveler** [reply]
Great!! Thanks

**Reply by TimSkytraveler** [reply]
Ahhh! It works perfect!! Thanks a lot.

**Comment by rmroc451** [comment]
what did you have for the unit on the button?  If you have nothing, and since it isn't one we assign a normal default (rpm, number, etc) it gets number.

Its probably a conversion issue.

Try putting in Percent in the unit field, and clicking out of that text box, see if that works.

If Percent isn't working, try "Rpm", and do the same clicking out of the box.

Also, I notice you have a semi colon inbetween the simconnect variable and the index, it should be a colon.

so what you put in the button should be this `TURB_ENG_FREE_TURBINE_TORQUE:1` to get the first engine (and only engine's) value.

**Comment by TimSkytraveler** [comment]
It just seems the Aerosoft Twin Otter does not output TURB_ENG_FREE_TURBINE_TORQUE (the semi colon was a typo here).
I am using this profile as a base:
https://flightsim.to/file/26525/twin-otter-stream-deck-profile-checklists#

**Comment by Gert57** [comment]
Hi Hy,

Thanks fort he response.

For the kodiak I try to retrieve the fuel flow in pounds per hour.

The kodiak developer said to use ENG_FUEL_FLOW_PPH:1

Units to use was      pounds per hour.

Hope you can do something.

Your flight tracker is splendid to use with streamdeck. It opens a whole new wordl for me retrieving info from the planes.

Makes flying a lot more immersive!!

 

As said: thanks fort he nice work!!

 

Sincerely yours,

 

Gert

 

Van: Nguyễn Quý Hy ***@***.***> 
Verzonden: woensdag 26 januari 2022 19:46
Aan: nguyenquyhy/Flight-Tracker-StreamDeck ***@***.***>
CC: Gert57 ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [nguyenquyhy/Flight-Tracker-StreamDeck] Free Turbine Torque (Discussion #328)

 

For now please give me the SimVar you need. I'm pretty slow with big features 😛.

—
Reply to this email directly,  <https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/discussions/328#discussioncomment-2053222> view it on GitHub, or  <https://github.com/notifications/unsubscribe-auth/AVYIEBIOCHBFWSDN5LW7ABTUYA6OLANCNFSM5MP2C3WA> unsubscribe.
Triage notifications on the go with GitHub Mobile for  <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> iOS or  <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> Android.

**Reply by nguyenquyhy** [reply]
Added and released in 0.41 (in GitHub Releases at the moment).

**Comment by Gert57** [comment]
Hi HY,

This is extra ordinary!!

Thank you very much!!

I will try it when I finish work in 20 minutes.

Will mail you if it works ok!!

But…. This kind of speed isn’t necessary. I really am able to wait a bit 😊

Maybe different from some in the msfs forums…..;-)

 

 

Sincere greetings,

 

Gert

 

Van: Nguyễn Quý Hy ***@***.***> 
Verzonden: woensdag 26 januari 2022 20:55
Aan: nguyenquyhy/Flight-Tracker-StreamDeck ***@***.***>
CC: Gert57 ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [nguyenquyhy/Flight-Tracker-StreamDeck] Free Turbine Torque (Discussion #328)

 

Added and released in 0.41 (in GitHub Releases at the moment).

—
Reply to this email directly,  <https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/discussions/328#discussioncomment-2053605> view it on GitHub, or  <https://github.com/notifications/unsubscribe-auth/AVYIEBJYW7H7JK6VZBQFL7TUYBGQTANCNFSM5MP2C3WA> unsubscribe.
Triage notifications on the go with GitHub Mobile for  <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> iOS or  <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> Android.

**Comment by Gert57** [comment]
Hi Hy,

It works ok!!

Thanks a lot!!

Cordial greettings!!

 

Gert

 

Van: Nguyễn Quý Hy ***@***.***> 
Verzonden: woensdag 26 januari 2022 20:55
Aan: nguyenquyhy/Flight-Tracker-StreamDeck ***@***.***>
CC: Gert57 ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [nguyenquyhy/Flight-Tracker-StreamDeck] Free Turbine Torque (Discussion #328)

 

Added and released in 0.41 (in GitHub Releases at the moment).

—
Reply to this email directly, view it on GitHub <https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/discussions/328#discussioncomment-2053605> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AVYIEBJYW7H7JK6VZBQFL7TUYBGQTANCNFSM5MP2C3WA> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .

**Comment by Gert57** [comment]
Hi Hy,

I donated $5 for your honeycomb bravo!

Thanks again!

See you in the virtual sky.

 

Gert

 

Van: Nguyễn Quý Hy ***@***.***> 
Verzonden: woensdag 26 januari 2022 20:55
Aan: nguyenquyhy/Flight-Tracker-StreamDeck ***@***.***>
CC: Gert57 ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [nguyenquyhy/Flight-Tracker-StreamDeck] Free Turbine Torque (Discussion #328)

 

Added and released in 0.41 (in GitHub Releases at the moment).

—
Reply to this email directly, view it on GitHub <https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/discussions/328#discussioncomment-2053605> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AVYIEBJYW7H7JK6VZBQFL7TUYBGQTANCNFSM5MP2C3WA> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .
